pages/index.js

```diff
@@ -1,7 +1,5 @@
+
+
 export default function HomePage() {
-  return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
-  );
+  return <div></div>;
```

Comment:
since you don't have any other pages it would be better to use your index.js instead of the `myart.js`. That way the user sees the content when they go directly to your home route.